index.html

```diff
@@ -1,224 +1,299 @@
+          <li><a class="contact-icon" href="#contact-detail"></a></li>
+        </ul>
+      </nav>
+      <!-- headline section -->
+      <section id="headline_section" class="headline_section">
+        <div>
+          <h1 class="h1">
+            Hey There.<br />
+            I'm Kifle
+          </h1>
 
-                </ul>
+          <h4 class="h4">I'm a Software Developer</h4>
 
-            </div>
-        </section>
+          <p class="p">
+            I can help you build a product , feature or website. Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
 
+        <!-- social media section -->
+        <ul class="image-icons-header">
+          <li>
+            <a href="https://github.com/kifle23"
+              ><img src="Images/GitHub.png" alt="GitHub"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/kifle-haile-5a613761"
+              ><img src="Images/Linkedin.png" alt="Linkedin"
+            /></a>
+          </li>
+          <li>
+            <a href="https://twitter.com/KifleHaile12"
+              ><img src="Images/Twitter.png" alt="Twitter"
+            /></a>
+          </li>
+        </ul>
+      </section>
     </header>
     <section id="resent-works">
-        <div class="heading-and-seperator">
-            <h1 class="heading">My Recent Works</h1><br />
-            <hr class="horizontal-seperator">
+      <div class="recent">
+        <div class="line">
+          <h2>My Recent Works</h2>
+          <hr />
         </div>
-        <div class="work-card-container">
-            <div class="img-one img"></div>
-            <h2 class="card-heading">Multi-Post Stories</h2>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
-                industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        <div class="img-txt">
+          <img src="Images/portfolio/card-one.svg" alt="" />
+          <div class="multi">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
             </p>
             <div class="skills">
-                <input type="button" value="CSS">
-                <input type="button" value="HTML">
-                <input type="button" value="Bootstrap">
-                <input type="button" value="Ruby">
+              <input type="button" value="CSS" />
+              <input type="button" value="HTML" />
+              <input type="button" value="Bootstrap" />
+              <input type="button" value="Ruby" />
```

Comment:
- [ ] Please make sure to follow the proper semantics ([more info here](https://www.freecodecamp.org/news/html-best-practices/)) as the `input` element should only be reserved for input-related content. ✔
- I suggest you use an `li` tag for each of the technologies and wrap them inside a `ul` tag.
- [ ] Also please make sure you follow the proper styling a shown in the [Figma design below](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=t45uZqimwMP5BSMR-0):
       ![image](https://user-images.githubusercontent.com/104968236/229129310-54c55655-d0f8-4ff0-a27c-518a02cf0e02.png)